src/sweeper/workspace_info.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# * coding: utf8 *
+'''
+workspace_info.py
+A module that gets information about the workspace, including a list of feature classes.
+'''
+import arcpy
+import os
+
+
+#: A function to generate and return a list of workspace feature classes.
+def get_featureclasses(workspace_path):
+    '''
+    workspace_path: full path to the feature workspace.
+    '''
+    arcpy.env.workspace = workspace_path
```

Comment:
this should be using the EnvManager like the sweeper classes do. Let's try to never set the `arcpy.env.workspace` directly.

Reply:
yeah, i don't like that either.  it seems like i have to based on this text, "The workspace environment must be set first before using several of the List functions" ??
https://pro.arcgis.com/en/pro-app/arcpy/functions/listfeatureclasses.htm

I'll try the arcpy.EnvManager(workspace=self.workspace): though and see if that works.

Reply:
@steveoh good catch.  Yes, this works: 
`with arcpy.EnvManager(workspace=workspace_path):`

is that what you're thinking?

Reply:
yup